src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js

```diff
@@ -304,6 +304,14 @@ function listViewController($scope, $interpolate, $routeParams, $injector, $time
                     navigationService.reloadSection(section);
                 }
             }
+        }, function(error){
+          // if someone attempts to add mix listviews across sections (i.e. use a members list view on content types),
+          // a not-supported exception will be most likely be thrown, at least for the default list views - lets be
+          // helpful and show a meaningful error message directly in content/content type UI
+          if(error.data.ExceptionType && error.data.ExceptionType.indexOf("System.NotSupportedException") > -1) {
```

Comment:
This seems a bit hacky, but probably not much to do in terms of recognizing the error on a deeper model layer except for looking for a specific string ;-)

Could you add an extra check for `error.data` as in `if (error.data && error.data.ExceptionType [...])`?

Reply:
Indeed hacky 😆 not my proudest moment, but I can't really see any other way around this very specific case.

Fixed in latest.